frontend/src/features/notes/list/NoteListView.tsx

```diff
@@ -0,0 +1,104 @@
+import GenericModal from 'components/common/GenericModal';
+import { TableSort } from 'components/Table/TableSort';
+import { NoteTypes } from 'constants/noteTypes';
+import { useApiNotes } from 'hooks/pims-api/useApiNotes';
+import useIsMounted from 'hooks/useIsMounted';
+import { orderBy } from 'lodash';
+import { Api_Note } from 'models/api/Note';
+import React from 'react';
+import { toast } from 'react-toastify';
+
+import { NoteResults } from './NoteResults/NoteResults';
+import * as Styled from './styles';
+
+export interface INoteListViewProps {
+  type: NoteTypes;
+}
```

Comment:
I think the ListView should take the entityId of the parent entity (activity or file) to show notes for that entity only